go_lib/deckhouse-config/status.go

```diff
@@ -147,12 +147,7 @@ func (s *StatusReporter) ForConfig(cfg *v1alpha1.ModuleConfig) ModuleConfigStatu
 	if chain.IsKnownVersion(cfg.Spec.Version) && hasVersionedSettings(cfg) {
 		res := Service().ConfigValidator().Validate(cfg)
 		if res.HasError() {
-			var prefix = "Ignored"
-			if s.moduleManager.IsModuleEnabled(cfg.GetName()) {
-				prefix = "Error"
-			}
-
-			invalidMsg := fmt.Sprintf("%s: %s", prefix, res.Error)
+			invalidMsg := fmt.Sprintf("Ignored: %s", res.Error)
```

Comment:
Shouldn't we have an `Error: %s` prefix here? Because we will have always enabled module here

Reply:
Fixed